drivers/rtc/rtc_ll_stm32.c

```diff
@@ -140,6 +141,35 @@ struct rtc_stm32_data {
 #endif /* STM32_RTC_ALARM_ENABLED */
 };
 
+#ifdef STM32_RTC_ALARM_ENABLED
+
+static inline void ll_func_exti_enable_rtc_alarm_it(uint32_t exti_line)
+{
+#if defined(CONFIG_SOC_SERIES_STM32U5X) || defined(CONFIG_SOC_SERIES_STM32WBAX)
+	/* in STM32U5 & STM32WBAX series, RTC Alarm event is not routed to EXTI */
+#else
+	int ret = 0;
+
+	ret = stm32_exti_enable(exti_line, STM32_EXTI_TRIG_RISING, STM32_EXTI_MODE_IT);
+	if (ret != 0) {
+		LOG_ERR("Failed to enable EXTI line %d", exti_line);
+	}
```

Comment:
Either we care about the return value, and it should be displayed:
```suggestion
	int ret;

	ret = stm32_exti_enable(exti_line, STM32_EXTI_TRIG_RISING, STM32_EXTI_MODE_IT);
	if (ret != 0) {
		LOG_ERR("Failed to enable EXTI line %d (error %d)", exti_line, ret);
	}
```
Or we don't care about it, and the following is sufficient:
```suggestion
	if (stm32_exti_enable(exti_line, STM32_EXTI_TRIG_RISING, STM32_EXTI_MODE_IT) != 0) {
		LOG_ERR("Failed to enable EXTI line %d", exti_line);
	}
```